fw/http_limits.c

```diff
@@ -342,15 +342,17 @@ frang_http_uri_len(const TfwHttpReq *req, FrangAcc *ra, unsigned int uri_len)
  * way to achieve that - is to embed a hook into HTTP-parsers (or in case of
  * HTTP/2, into HPACK-decoder) directly to catch the long headers immediately.
  */
+
 static int
 __frang_http_field_len(const TfwHttpReq *req, FrangAcc *ra,
 		       unsigned int field_len,  unsigned int hdr_cnt)
 {
 	const TfwStr *field, *end, *dup, *dup_end;
 
-	if (hdr_cnt && req->h_tbl->off >= hdr_cnt) {
-		frang_limmsg("HTTP headers number", req->h_tbl->off,
+	if (hdr_cnt && req->h_tbl->cnt > hdr_cnt) {
+		frang_limmsg("HTTP headers number", req->h_tbl->cnt,
 			     hdr_cnt, &FRANG_ACC2CLI(ra)->addr);
+
```

Comment:
Here and in line 345 are senseless changes of just adding 2 empty lines

Reply:
Fixed